CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### [3.0.3] - 2025-06-02
+
+### Added
+
+- Default Language management: Changes needed to make nuxt sections library able of configuring the defaultlocale of the nuxt/i18n library configuration #112
+- New tab in the settings popup to customize css settings available in the Sections css preset #180
```

Comment:
In this library, the change is to provide the ability for the host project to define a form that will be displayed dynamically in the settings.

It can be one or multiple tabs and the purpose of the form exposed is not controlled by the library